src/ed_object_models/conversion_sdf.py

```diff
@@ -253,8 +305,15 @@ def read_shape(shape: Mapping, link_names: List[str], color: OrderedDict, model_
     """
     sdf_link = []
     # Check if compound type has name (inserted as comment in yaml)
-    if "compound" in shape:
-        for item in shape["compound"]:
+    keys = [key for key in ["compound", "group"] if key in shape]
```

Comment:
I don't think its a good idea to support two formats for this function. What new files use 'group' and why cant they use 'compound'

Reply:
This has always been in the cpp code of ed. The convertor was therefore incomplete.